src/pages/api/user/register.ts

```diff
@@ -15,7 +15,9 @@ const registrationHandler = async (
     return;
   }
 
-  const result = registrationSchema.safeParse(req.body);
+  const body = JSON.parse(req.body);
```

Comment:
This shouldn't be required. I can send requests through Postman without having to sent the request body as a string.